website/cue/reference/components/sources/http_client.cue

```diff
@@ -51,84 +52,7 @@ components: sources: http_client: {
 		platform_name: null
 	}
 
-	configuration: {
-		auth: configuration._http_auth & {_args: {
-			password_example: "${PASSWORD}"
-			username_example: "${USERNAME}"
-		}}
-		endpoint: {
-			description: "Endpoint to collect observability data from."
-			required:    true
-			warnings: ["You must explicitly add the path to your endpoint."]
```

Comment:
I asked about this offline. Flagging here to not forget. I don't think we have a way to add this with the configurable macros.

Reply:
Noting that @StephenWakely seemingly handled it like [so](https://github.com/vectordotdev/vector/pull/15901/files/56c1b3d2afd155a5fb8e5ff1ff33b0b489dbb1fc..900f277206b4a389808f0fd14f46189e0ea0e478#diff-667cf782642616aecc46a83c6630f7c4edb2a7a50a19108218ee30e609c47041R48-R60) - worth a doublecheck that this is the way to go though.

Reply:
added here: https://github.com/vectordotdev/vector/pull/16054/commits/c0e1610e401b3e675ca6f0b63ed560de5fea6f44

Reply:
Yeah, that's the only way we can add warnings right now.